craft_providers/actions/snap_installer.py

```diff
@@ -340,7 +340,13 @@ def inject_from_host(*, executor: Executor, snap_name: str, classic: bool) -> No
             classic,
         )
 
-    host_revision = get_host_snap_info(snap_name)["revision"]
+    host_snap_info = get_host_snap_info(snap_name)
+    host_snap_base = host_snap_info.get("base", None)
+    if host_snap_base:
+        logger.debug("Installing base snap %r from host", host_snap_base)
```

Comment:
This may add useful context to the logs:

```suggestion
        logger.debug("Installing base snap %r for %r from host", host_snap_base, snap_name)
```